cogs/games.py

```diff
@@ -58,10 +60,151 @@ async def ball8(self, ctx: Context, *, question: str) -> None:
         elif reply_type == 3:
             answer = random.choice(UNSURE_BALL8_RESPONSES)
 
-        embed = Embed(title="Magic 8-ball", color=Color.blurple(),)
+        embed = Embed(title="Magic 8-ball", color=Color.blurple())
         embed.add_field(name="Question", value=question)
         embed.add_field(name="Answer", value=answer)
 
+    @command()
+    async def hangman(self, ctx: Context) -> None:
+        def display_hangman(tries: int) -> str:
+            stages = [  # final state: head, torso, both arms, and both legs
+                r"""
```

Comment:
This comment should go above r""" not besides the bracket
```suggestion
            stages = [  
                # final state: head, torso, both arms, and both legs
                r"""
```